includes/functions/metadata.php

```diff
@@ -292,7 +292,7 @@ function resource_data_init() {
 
 	$cmb->add_field(
 		[
-			'name'        => __( 'Permanent Link', 'learning-commons-framework' ),
+			'name'        => __( 'Permanent Link (Required)', 'learning-commons-framework' ),
```

Comment:
It was easy to miss the "(Required)" text as it is indistinct from the other text in the label.

Adding an asterisk next to the input field may help visually draw attention to it. Using the `::after` pseudo-element for the input field can work as we don't want it being read by an AT (since the text "required" is already in the label).